apps/files/src/components/FileList.vue

```diff
@@ -200,21 +207,32 @@ export default {
     },
 
     $_actionInProgress (item) {
-      return this.inProgress.some(itemInProgress => itemInProgress.id === item.id)
+      return this.actionsInProgress.some(itemInProgress => itemInProgress.id === item.id)
     },
 
     $_disabledActionTooltip (item) {
       if (this.$_actionInProgress(item)) {
-        const message = this.$gettext('Another action is currently in progress for this %{itemType}')
-        return this.$gettextInterpolate(message, { itemType: item.type })
+        if (item.type === 'folder') {
+          return this.$gettext('Another action is currently in progress for this folder')
+        }
+
+        return this.$gettext('Another action is currently in progress for this file')
       }
 
       return null
+    },
+
+    $_disabledActionSpinnerTooltip (item) {
+      if (item.type === 'folder') {
+        return this.$gettext('There is currently an action in progress for this folder')
```

Comment:
can we reuse the message as above? We shall try to reuse existing messages as far as possible. This will reduce workload for translators.

Reply:
Sure, I'll adjust it.

Reply:
Done